README.md

```diff
@@ -31,21 +31,22 @@ What does it do?
 <sup>² Only supported on Travis</sup><br>
 <sup>³ Beta support until Travis CI fixes <a href="https://travis-ci.community/t/no-space-left-on-device-for-system-z/5954/11">a bug</a></sup><br>
 
-- Builds manylinux, macOS and Windows wheels for CPython and PyPy using Azure Pipelines, Travis CI, AppVeyor, and CircleCI
+- Builds manylinux, macOS, and Windows wheels for CPython and PyPy
+- Works on GitHub Actions, Azure Pipelines, Travis CI, AppVeyor, CircleCI, and GitLab CI
 - Bundles shared library dependencies on Linux and macOS through [auditwheel](https://github.com/pypa/auditwheel) and [delocate](https://github.com/matthew-brett/delocate)
 - Runs the library test suite against the wheel-installed version of your library
 
 Usage
 -----
 
-`cibuildwheel` currently works on **Travis CI**, **Azure Pipelines**, **AppVeyor**, **GitHub Actions**, **CircleCI**, and **Gitlab CI**. Check the table below for supported platforms on each service:
+`cibuildwheel` currently works on **GitHub Actions**, **Azure Pipelines**, **Travis CI**, **AppVeyor**, **CircleCI**, and **GitLab CI**. Check the table below for supported platforms on each service:
```

Comment:
This reference seems especially redundant, given the info is in the table below.
```suggestion
`cibuildwheel` runs inside a CI service. Supported platforms depend on which service you're using:
```